### PR TITLE
Fix contact create method url

### PR DESCRIPTION
**Pull request by karaxuna**
Change-type: minor
Connects-to: #36

### CODE REVIEW

File: build/index.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ var Front = (function () {
                 return _this.httpCall({ method: 'GET', path: 'conversations' }, null, callback);
             },
             update: function (params, callback) {
-                return _this.httpCall({ method: 'PATCH', path: 'conversations/<conversation_id>' }, params, callback);
```
Comment (sradevski) [inline build/index.js:73]: Is this change necessary? Looking from the rest of the code, it seems the `<>` will be replaced with the parameter, doesn't it work like that?

Comment (karaxuna) [reply build/index.js:73]: I have not made that change, this is built javascript file. Somebody who changed this must have forgotten to build, so it appeared as change on my build.

**Review by sradevski — APPROVED** [review]
LGTM

File: build/index.js (hunk 1 above)
Comment (sradevski) [reply build/index.js:73]: I see, anyway it will work both ways, so it's fine with me.

Comment (karaxuna) [reply build/index.js]: Wait wait...

Comment (karaxuna) [reply build/index.js:73]: Here: https://github.com/balena-io-modules/front-sdk/commit/0f6d452a2e87d23ecf28181238ca767ffb59d014#diff-13b5b151431c7e7a17f273559ed212d5

**Comment by karaxuna** [comment]
@balena-ci retest
